frontend-plugin-core/src/test/java/com/github/eirslett/maven/plugins/frontend/lib/UtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package com.github.eirslett.maven.plugins.frontend.lib;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Since you only use arraylist you can initialise everything as arraylists and remove the list import